Source/WebCore/rendering/style/StyleRareNonInheritedData.h

```diff
@@ -213,7 +213,8 @@ class StyleRareNonInheritedData : public RefCounted<StyleRareNonInheritedData> {
 
     unsigned textDecorationStyle : 3; // TextDecorationStyle
 
-    unsigned aspectRatioType : 3;
+    unsigned aspectRatioType : 2;
```

Comment:
Nice fix; I wonder if there’s a good way for us to check that all these bitfields are the correct size. Clearly we got it wrong doing it by hand.

Separately, maybe we should use the convention here where we write the name of the enumeration in a comment as is done above with `TextDecorationStyle`. But obviously that does not guarantee. we check the sizes.

Reply:
To be fair, we (Igalia) introduced the bitfield that is too big, so only right to fix it. Will add the comment.